gcapi/client.py

```diff
@@ -833,9 +833,22 @@ def add_case_to_reader_study(
         This function takes a reader-study slug and a list of socket value specs.
         It then creates a single display set for the reader study.
 
+        Example:
+            ```Python
+            from gcapi import SocketValueSpec
+
+            client.add_case_to_reader_study(
+                reader_study_slug="i-am-a-reader-study",
+                values=[
+                    SocketValueSpec(socket_slug="report", files=["report.pdf"]),
+                    SocketValueSpec(socket_slug="lung-volume", value=1.9),
+                ],
+            )
+            ```
+
         ??? tip "Re-using existing images"
             Existing images on Grand Challenge can be re-used by either
-            passing an API url, or an existingsocket value (from a display set):
+            passing an API url, or an existing socket value (from a display set):
```

Comment:
```suggestion
            passing an API url, or an existing socket value: 
```
It can be from any item, right? Same with archive items, somewhere below.

Reply:
Ah True!